src/util.rs

```diff
@@ -115,6 +117,9 @@ pub fn split_glob_on_match<'a, 'b>(glob_pattern: &'a str, shv_path: &'b str) ->
     if pattern.matches(trimmed_path) {
         match globstar_pos {
             None => {
+                // We don't probably want to use `cmp()` and match, as it might be slower:
+                // https://rust-lang.github.io/rust-clippy/master/index.html#/comparison_chain
+                #[allow(clippy::comparison_chain)]
```

Comment:
replace exhausting `if chain` with match

Reply:
According to https://rust-lang.github.io/rust-clippy/master/index.html#/comparison_chain `match`  may have performance loss